### DISCUSSION TITLE
Moveit2 build failed  in windows11 and ros2-foxy

### DISCUSSION THREAD

**Discussion by miaoccy** [moveit/moveit2#2379] [Q&A]
`C:\opt\ros\moveit2_ws>colcon build --packages-skip-by-dep %IGNORED_PACKAGES%  --packages-skip %IGNORED_PACKAGES% --event-handlers desktop_notification- status- --cmake-args -D CMAKE_BUILD_TYPE=Release
[1.008s] colcon.colcon_core.package_selection WARNING ignoring unknown package 'warehouse_ros_mongo' in --packages-skip
[1.373s] root DEBUG Using proactor: IocpProactor
Starting >>> moveit_common
[1.390s] colcon.colcon_ros.prefix_path.catkin WARNING The path 'C:\opt\ros\foxy\x64\tools\vcpkg\installed\x64-windows' in the environment variable CMAKE_PREFIX_PATH doesn't exist
Finished <<< moveit_common [3.67s]
Starting >>> moveit_core
[Processing: moveit_core]
Finished <<< moveit_core [37.8s]
Starting >>> moveit_ros_occupancy_map_monitor
Starting >>> moveit_simple_controller_manager
Starting >>> chomp_motion_planner
Finished <<< moveit_simple_controller_manager [14.3s]
Starting >>> moveit_plugins
Finished <<< moveit_ros_occupancy_map_monitor [15.0s]
Starting >>> moveit_ros_planning
Finished <<< chomp_motion_planner [17.3s]
Starting >>> moveit_chomp_optimizer_adapter
Starting >>> moveit_planners_chomp
Finished <<< moveit_plugins [5.92s]
Failed   <<< moveit_chomp_optimizer_adapter [16.0s, exited with code 1]
Aborted  <<< moveit_planners_chomp [17.1s]
Aborted  <<< moveit_ros_planning [28.2s]

Summary: 6 packages finished [1min 25s]
  1 package failed: moveit_chomp_optimizer_adapter
  2 packages aborted: moveit_planners_chomp moveit_ros_planning
  17 packages not processed`

I'm installed ros2 is Binary  ros-foxy-desktop Metapackage. But ,i'm not find Binary  moveit 2

**Comment by bmaxdk** [comment]
The warning shows here `CMAKE_PREFIX_PATH` suggests that there might be an issue with your environment setup. Make sure that the path in `CMAKE_PREFIX_PATH` exists and correctly set up. This path should point to the directories where your dependencies are installed.
